brian_cheung/the_basics.md

```diff
@@ -160,5 +168,9 @@ end
 
 **Answer 12:**
 ```ruby
-# write your code here
+def Caps(string)
+	puts "#{string}".upcase
+end
+
+
```

Comment:
A couple of things:
1. Seems like a nitpick, but Ruby methods are typically not capitalized
2. Nitpick: Watch that tabbing on line 172 - it's pretty far over. Check your Sublime settings in the bottom right corner. I generally set it so I indent using spaces, with a tab width of 2.
3. No need to interpolate the string here, since it's already a string - you can just write `string.upcase`
4. How would you call this method?